src/lib/media/picker.e2e.tsx

```diff
@@ -13,7 +13,7 @@ async function getFile() {
       .join('/'),
   )
   const file = files[_imageCounter++ % files.length]
-  return await compressAndResizeImageForPost({
+  return await compressIfNeeded({
```

Comment:
@pfrazee I realize this may not be in line with the original intent but looking at the function, it seems sufficient since we just want **any** file to use with tests.